docs/superpowers/specs/2026-05-06-realtime-party-chat-design.md

```
@@ -0,0 +1,218 @@
+| 사용자 유형 | 인증 수단 |
+|------------|----------|
+| 로그인 사용자 | `Authorization: Bearer {jwt}` |
+| 비로그인 사용자 | `X-Participant-Token: {participantToken}` |
+
+두 헤더가 모두 없으면 `UNAUTHORIZED` 에러.
+
+## API
+
+### 1. 라이브 입장 (닉네임 + 캐릭터 선택)
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)